stack/database.py

```diff
@@ -117,8 +144,8 @@
     DBInstanceClass=Ref(db_class),
     DBInstanceIdentifier=Ref(AWS_STACK_NAME),
     Engine="postgres",
-    EngineVersion="9.4.5",
-    MultiAZ=True,
+    EngineVersion=Ref(db_engine_version),
+    MultiAZ=Ref(db_multi_az),
```

Comment:
My understanding is that MultiAZ triggers a synchronous master/slave set up. So based on the docs you quoted, the latency is actually higher with MultiAZ, because it has to wait for the write to succeed in both instances instead of just one. DNS is only used for failover.